helm/charts/cloud-carbon-footprint/README.md

```diff
@@ -0,0 +1,119 @@
+<!--- app-name:  cloud-carbon-footprint -->
+
+# Helm Chart for Cloud Carbon Foot Print
+
+Cloud Carbon Footprint is an application that estimates the energy (kilowatt hours) and carbon emissions (metric tons CO2e) of public cloud provider utilization.
+
+
+## TL;DR
+
+```console
+git clone https://github.com/appnetwise/cloud-carbon-footprint.git
+cd helm/charts/cloud-carbon-footprint
+helm install cloud-carbon-footprint .
+```
+
+## Introduction
+
+[Cloud Carbon Footprint](https://www.cloudcarbonfootprint.org) is an application that estimates the energy (kilowatt hours) and carbon emissions (metric tons CO2e) of public cloud provider utilization.
+
+If you would like to learn more about the various calculations and constants that we use for the emissions estimates, check out the [Methodology page](https://www.cloudcarbonfootprint.org/docs/methodology)
+
+
+
+## Prerequisites
+
+- Kubernetes
+- Helm 
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```console
+helm install my-release .
+```
+
+## Parameters
+### Client Configuration
+
+| Parameter                                   | Description                                                                            | Default Value                                                                            |
+|---------------------------------------------|----------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------|
+| `client.configmap.resolver`                 | Resolver configuration for the client                                                  | `""`                                                                                     |
+| `client.deployment.replicaCount`            | Number of replicas for the client deployment                                           | `2`                                                                                      |
+| `client.deployment.image.repository`        | Docker image repository for the client                                                 | `docker.io/cloudcarbonfootprint/client`                                                  |
+| `client.deployment.image.tag`               | Docker image tag for the client                                                        | `release-2022-10-17@sha256:2662253ad7589efa1eb2e8f84b55d2e3ff11eb4a218e1ff377ab31a0b8de763d` |
```

Comment:
I believe you updated default value was to release-2024-02-11

Reply:
Yes updated in the readme file